config/pipelines/Jenkinsfile-contra-lib-sample/JenkinsfileKojiTaskBuild

```diff
@@ -0,0 +1,86 @@
+CANNED_CI_MESSAGE = '{"build_id":1104742,"old":0,"name":"krb5","task_id":28114680,"attribute":"state","request":["git+https://src.fedoraproject.org/rpms/krb5.git#40a05d03472a1bcc7aa57f32273357732b2fe895","f28-candidate",{}],"instance":"primary","epoch":null,"version":"1.16.1","owner":"rharwood","new":1,"release":"10.fc28","topic":"org.fedoraproject.prod.buildsys.build.state.change"}'
+
+def libraries = ['contra-lib': ['jmsMessages', 'https://github.com/joejstuart/contra-lib.git']]
+
+libraries.each { name, repo ->
+    library identifier: "${name}@${repo[0]}",
+            retriever: modernSCM([$class: 'GitSCMSource',
+                                  remote: repo[1]])
+
+}
+
+properties(
+        [
+                buildDiscarder(logRotator(artifactDaysToKeepStr: '30', artifactNumToKeepStr: '', daysToKeepStr: '90', numToKeepStr: '')),
+                disableConcurrentBuilds(),
+                parameters(
+                        [
+                                string(defaultValue: "${env.ghprbActualCommit}", description: '', name: 'ghprbActualCommit'),
+                                string(defaultValue: "${env.ghprbGhRepository}", description: '', name: 'ghprbGhRepository'),
+                                string(defaultValue: '', description: '', name: 'ghprbPullId'),
+                                string(defaultValue: '', description: '', name: 'ghprbPullAuthorLogin'),
+                                string(defaultValue: 'continuous-infra', description: 'Project namespace for Openshift operations', name: 'OPENSHIFT_NAMESPACE'),
+                                string(defaultValue: '172.30.1.1:5000', description: 'The internal docker registry url', name: 'DOCKER_REPO_URL'),
+                                string(defaultValue: 'stable', description: 'The rpmbuild image tag', name: 'RPMBUILD_TAG'),
+                                string(defaultValue: 'stable', description: 'The cloud-image-compose image tag', name: 'CLOUD_IMAGE_COMPOSE_TAG'),
+                                string(defaultValue: 'stable', description: 'The singlehost-test image tag', name: 'SINGLEHOST_TEST_TAG'),
+                                string(defaultValue: "${CANNED_CI_MESSAGE}", description: 'Default ci message', name: 'CI_MESSAGE')
+
+                        ]
+                )
+        ]
+)
+
+containers = [
+        'rpmbuild':
+        [
+                tag: env.RPMBUILD_TAG
```

Comment:
I don't think any of these tags are defined in env

Reply:
Noted. Changed to latest. thanks